### ISSUE TITLE
Understanding the loss function for FullSupportBarDistribution

### ISSUE THREAD

**Issue by pg2455** [automl/PFNs#3]
Hi, 

I have been trying to understand the loss function for the regression case. The [code for FullBarDistriution.forward Line 251](https://github.com/automl/PFNs/blob/main/pfns/bar_distribution.py#L251) has mean_prediction_logits as its parameter. 

I couldn't find any reference to this in the main paper. Apologies if I missed it! Could you please explain what is it doing and whether it is necessary to do so? My concern is also regarding the mismatch of the dimensions, i.e., [Line 257](https://github.com/automl/PFNs/blob/main/pfns/bar_distribution.py#L257) require nll_loss and smooth_loss to be of the same dimension but it might not be possible after Line 251.

**Comment by SamuelGabriel** [comment]
it might just be a naming issue: in the paper we call it "riemann distribution", that is the same as the full bar distribution.

**Comment by pg2455** [comment]
This didn't really answer my question, but I am able to make it work without needing any mean_prediction_logits.
